Libs/Visualization/VTK/Widgets/ctkVTKDiscretizableColorTransferWidget.h

```diff
@@ -70,6 +70,11 @@ class CTK_VISUALIZATION_VTK_WIDGETS_EXPORT ctkVTKDiscretizableColorTransferWidge
   void copyColorTransferFunction(vtkScalarsToColors* ctf, bool useCtfRange = false);
   vtkDiscretizableColorTransferFunction* discretizableColorTransferFunction() const;
 
+  /// Set/Get the left axis mode, which controls the axis range computation.
+  /// See vtkScalarsToColorsHistogramChart for more details.
+  void setLeftAxisMode(int mode);
+  int getLeftAxisMode();
```

Comment:
`getLeftAxisMode` -> `leftAxisMode`

Reply:
Add a property like it is done here: https://github.com/tcoulange/CTK/blob/326c74e11061c06cd6484415c366096228f7aa8c/Libs/Visualization/VTK/Widgets/ctkVTKDiscretizableColorTransferWidget.h#L47-L50